.github/workflows/pytest_benchmarks.yml

```diff
@@ -3,9 +3,14 @@
 
 name: benchmark
 
+#on:
+#  push:
+#    branches: ["master", "release-*"]
 on:
   push:
     branches: ["master", "release-*"]
+  pull_request:
```

Comment:
Now that you've verified that the benchmark tests work with your change, is the plan to revert these files before merging the PR?

Reply:
The plan is to add them to the main test suite (`pytest.yml`) and remove `pytest_benchmarks.yml`